app/sass/app.scss

```diff
@@ -9,4 +9,5 @@
 @import 'atomic.scss';
 @import 'selectOptions.scss';
 @import 'sql.scss';
-@import 'print.scss';
\ No newline at end of file
+@import 'print.scss';
+@import 'joint.scss';
```

Comment:
Can we rename it to `joint-custom.scss`?
This way we can make it clear this file is not part of _joint core_

Reply:
Done!